sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/RemoveRedundantAliasAndProjectSuite.scala

```diff
@@ -40,8 +45,8 @@ class RemoveAliasOnlyProjectSuite extends PlanTest with PredicateHelper {
 
   test("all expressions in project list are aliased child output but with different order") {
     val relation = LocalRelation('a.int, 'b.int)
-    val query = relation.select('b as 'b, 'a as 'a).analyze
-    val optimized = Optimize.execute(query)
+    val query = relation.select('b, 'a).analyze
+    val optimized = Optimize.execute(relation.select('b as 'b, 'a as 'a).analyze)
```

Comment:
according to most of the optimizer unit tests, the preferred code style should be
```
val query = relation.select('b as 'b, 'a as 'a).analyze
val optimized = Optimize.execute(query)
val expected = relation.select('b, 'a).analyze
comparePlans(optimized, expected)
```